card/spec/models/card/create_spec.rb

```diff
@@ -3,7 +3,7 @@
 # FIXME this shouldn't be here
 describe Card::Set::Type::Cardtype, ".create with :codename" do
   it "should work" do
-    expect(Card.create!(:name=>"Foo Type", :codename=>"foo", :type=>'Cardtype').type_code).to eq(:cardtype)
+    expect(Card.create!(name: "Foo Type", codename: "foo", type: 'Cardtype').type_code).to eq(:cardtype)
```

Comment:
Line is too long. [104/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.